op-challenger/game/fault/trace/cannon/provider.go

```diff
@@ -168,6 +184,9 @@ func (p *CannonTraceProvider) loadProof(ctx context.Context, i uint64) (*proofDa
 				if err != nil {
 					return nil, fmt.Errorf("cannot hash witness: %w", err)
 				}
+				if err := WriteLastStep(p.dir, p.lastStep+1); err != nil {
```

Comment:
Could also use `state.Step` - I had to dig a little to figure out why 1 was being added to `p.lastStep`